src/zcl_abapgit_file_status.clas.abap

```diff
@@ -213,6 +213,14 @@ CLASS zcl_abapgit_file_status IMPLEMENTATION.
     lt_remote    = it_remote.
     SORT lt_remote BY path filename.
 
+    " Skip ignored files
+    LOOP AT lt_remote ASSIGNING <ls_remote>.
+      IF io_dot->is_ignored( iv_path     = <ls_remote>-path
+                             iv_filename = <ls_remote>-filename ) = abap_true.
+        DELETE lt_remote.
```

Comment:
reading the code, I'm not exactly sure what this statement does without testing it, the previous code deleted a specific row, if the intention is to delete the contents then I suggest using CLEAR

Reply:
`DELETE itab` inside a loop deletes the row, see [link](https://help.sap.com/doc/abapdocu_752_index_htm/7.52/en-us/abapdelete_itab_index.htm#!ABAP_ONE_ADD@2@)

Reply:
okay, from the same help "_This alternative is not recommended. Instead, use the addition INDEX to specify the row number explicitly._"